cranelift/docs/ir.md

```diff
@@ -311,23 +311,23 @@ Signaling NaNs
 
 ## Control flow
 
-Branches transfer control to a new EBB and provide values for the target EBB's
+Branches transfer control to a new BB and provide values for the target BB's
 arguments, if it has any. Conditional branches only take the branch if their
 condition is satisfied, otherwise execution continues at the following
-instruction in the EBB.
+instruction in the BB.
 
-JT = jump_table [EBB0, EBB1, ..., EBBn]
+JT = jump_table [BB0, BB1, ..., BBn]
     Declare a jump table in the [function preamble].
```

Comment:
We don't declare jump tables in the function preamble any more either, do we?

Reply:
I just removed that after noticing the same thing by @cfallin's comment :D